daphne/dapf/src/bin/dapf.rs

```diff
@@ -22,25 +22,25 @@ use url::Url;
 /// DAP Functions, a utility for interacting with DAP deployments.
 #[derive(Parser, Debug)]
 #[clap(version, about, long_about = None)]
-struct Cli {
+pub struct Cli {
```

Comment:
This no longer needs to be public, right? (Here and below.)

Reply:
Yes, you are right.